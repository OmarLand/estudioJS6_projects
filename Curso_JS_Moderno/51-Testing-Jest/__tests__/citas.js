import Citas from "../js/classes/Citas";

describe('Probar la clave de citas', () => {

    const citas = new Citas();

    test('Agregar nueva cita', () => {
        const citaObj = {
            mascota : 'Pelosa',
            propietario : 'Omar',
            telefono : '123456789',
            fecha : '15/12/2024',
            hora : '23:12',
            sintomas : 'Esta enfermo'
        };

        citaObj.id = Date.now();
        citas.agregarCita( citaObj );

        // Prueba
        expect(citas).toMatchSnapshot();
    });

});