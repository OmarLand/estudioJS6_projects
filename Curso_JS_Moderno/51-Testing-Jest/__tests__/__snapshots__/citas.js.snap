// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Probar la clave de citas Agregar nueva cita 1`] = `
Citas {
  "citas": [
    {
      "fecha": "15/12/2024",
      "hora": "23:12",
      "id": 1734301015779,
      "mascota": "Pelosa",
      "propietario": "Omar",
      "sintomas": "Esta enfermo",
      "telefono": "123456789",
    },
  ],
}
`;
